package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A minimal boilerplate for running node.js with babel",
   "main": "dist/index.js",
   "scripts": {
-    "start": "node dist/index.js",
+    "start": "nodemon dist/index.js",
```

Comment:
@MohammadAbd2 you'll need to do `npm install` to get the extra dependency i added.